src/logic/operate.js

```diff
@@ -0,0 +1,36 @@
+import Big from 'big.js';
+
+const operate = (numberOne, numberTwo, operation) => {
+  let res;
+  const firstArg = new Big(numberOne);
+  const secondArg = new Big(numberTwo);
+
+  switch (operation) {
+    case '-':
+      res = firstArg.minus(secondArg);
+      break;
+    case '÷':
+      if (secondArg !== 0) {
+        res = firstArg.div(secondArg);
+      } else {
+        res = 'Math Error';
+      }
+      break;
+    case 'x':
+      res = firstArg.times(secondArg);
+      break;
+    case '+':
+      res = firstArg.plus(secondArg);
+      break;
+    case '%':
+      res = firstArg.div(100);
```

Comment:
- [ ] Instead of using `div` here, try using the [`mod`](http://mikemcl.github.io/big.js/#mod) function from `Big.js` module to work with percentages.